dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -49,13 +49,16 @@ def check_consistency(self):
         self._check_dataset_time_consistency(self._load_data)
         self._check_dataset_internal_consistency()
 
-    @track_timing(timer_stats_collector)
-    def get_unique_dimension_rows(self):
-        """Get distinct combinations of remapped dimensions, including id.
-        Check each col in combination for null value."""
-        dim_table = self._remap_dimension_columns(self._load_data_lookup, False).distinct()
-        check_null_value_in_unique_dimension_rows(dim_table)
-        return dim_table
+    def make_dimension_association_table(self) -> DataFrame:
+        df = self._remap_dimension_columns(self._load_data_lookup, False).distinct()
```

Comment:
Do we need to filter out id and scaling factor here?

Reply:
Those will get ignored later, but we should do that to minimize the size. I’ll add a function to do this for both handlers.